Update values (#112)
abtion/guidelines: behaviour_and_protocol/values.md

> # Collective ownership

Reviewer: ```suggestion
Values are pointless if not put into practice. Our values are reflected in the following behaviors.

# Collective ownership
```

Collective ownership gives control of the entire project to all team members. "Your part" is the product. Either it's a good product, or it is not. 

## Collective vision

> - Open communication: No privileged information.

Reviewer 2: To follow the same structure through all of them:
```suggestion
- Open communication. No privileged information.
```

> - Open communication: No privileged information.

Reviewer: ```suggestion
- Communicate openly. No privileged information.
```

Making this an imperative, more action-oriented.

- Cut the middle person. We put the team directly in front of the stakeholder.
- Break knowledge silos. No one is "the expert" in an area.

## Collective accountability
- Work end-to-end. We need to be able to impact as much of the project as possible.

> - T-shape profiles. We still have roles, and they know about areas surrounding their expertise.

Reviewer 2: It seems like the adjective "T-shaped" is more common:
```suggestion
- T-shaped profiles. We still have roles, and they know about areas surrounding their expertise.
```

Reviewer 2: Could this be something like:
`T-shaped profiles. We each have strengths, but we understand areas outside of our expertise.`

Author: I feel that this one needs another turn. I feel "strengths" is a too vague word. 
I'm going to keep it out of this PR, but feel free to open a new one or just go ahead, pair up with someone and improve the wording.

> - Integrated Q&A. The team takes direct responsibility for the work they've created.

Reviewer 2: ```suggestion
- Integrated QA. The team takes direct responsibility for the work they've created.
```

Reviewer 3: In 5 years it will just be Q, and later it will probably be an emoji.

> - Integrated Q&A. The team takes direct responsibility for the work they've created.

Reviewer: ```suggestion
- Integrated QA. The team takes direct responsibility for the work they've created.
```

I think Q&A means questions and answers, whereas QA means quality assurance.

## Collective contribution

> - Pairing. Teams create the

Reviewer 4: @hcarreras truncated sentence here :)

Author: An issue when copying/pasting. Thanks for the heads up

> - Pairing. Team members work simultaneously on the same tasks.

Reviewer: ```suggestion
- Pair up! Team members work simultaneously on the same tasks.
```
